src/sdk/sdk.ts

```diff
@@ -98,31 +100,51 @@ export class FusionSDK {
             permit: params.permit
         })
 
+        const domain = getLimitOrderV3Domain(this.config.network)
+        const hash = order.getOrderHash(domain)
+
+        return {order, hash, quoteId: quote.quoteId}
+    }
+
+    public async submitOrder(
+        walletAddress: string,
+        quoteId: string,
+        order: FusionOrder
```

Comment:
1. Remove `walletAddress`. Use `orderStruct.maker` instead
2. Change direction to `order: FusionOrder, quoteId: string`